src/Grid/grid.jl

```diff
@@ -13,10 +13,23 @@ struct Node{dim,T}
     x::Vec{dim,T}
 end
 Node(x::NTuple{dim,T}) where {dim,T} = Node(Vec{dim,T}(x))
+
+"""
+    getcoordinates(::Node)
+    
+Get the value of the node coordinate.
+"""
 getcoordinates(n::Node) = n.x
 
 """
-    Ferrite.get_coordinate_eltype(::Node)
+    get_coordinate_type(::Node)
+
+Get the data type of the the node coordinate.
+"""
+get_coordinate_type(::Node{dim,T}) where {dim,T}  = Vec{dim,T}
```

Comment:
Perhaps this should be in the AbstractGrid interface, as we discussed in #579 ?

Reply:
Good point.